### PR TITLE
[16.0][MIG] sale_timesheet_task_exclude

### PR DESCRIPTION
**Pull request by Urvisha-OSI**

### CODE REVIEW

File: sale_timesheet_task_exclude/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2023 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (CRogos) [inline sale_timesheet_task_exclude/__init__.py]: Are changes of the Copyright necessary, even though you did not change any code?!?

File: sale_timesheet_task_exclude/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# Copyright 2020 CorporateHub (https://corporatehub.eu)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sales Timesheet: exclude Task from Sale Order",
+    "version": "16.0.1.0.1",
```
Comment (CRogos) [inline sale_timesheet_task_exclude/__manifest__.py]: Version should be 16.0.1.0.0

**Review by CRogos — CHANGES_REQUESTED** [review]
Could you squash all changes in the [MIG] commit?

**Comment by rafaelbn** [comment]
Please change name of PR with exact name of module s`ale_timesheet_line_exclude` in order to reviewers to find it 😄

**Comment by CRogos** [comment]
> Please change name of PR with exact name of module s`ale_timesheet_line_exclude` in order to reviewers to find it 😄

I think you mean sale_timesheet_**task**_exclude ;-)

**Comment by rafaelbn** [comment]
Yes! Sorry! `sale_timesheet_task_exclude` Thank you! 😄 

https://github.com/OCA/timesheet/tree/15.0/sale_timesheet_task_exclude
https://github.com/OCA/timesheet/tree/15.0/sale_timesheet_line_exclude

**Comment by CRogos** [comment]
Thanks for the quick response and improvement. Could you sqash:
image

There are still changes on the Copyright headers. I would prefer to leave these files unchanged. Not all occurrences are marked, these are just examples, but you can easily clean it up when you squash the commit and review all changes.
image

**Comment by CRogos** [comment]
A lot better, but now miss the commits as you see it from the v13 migration.
image
In this case, the first commit should contain these two files, while the rest can go to the second commit.
image

Everything else looks perfect.

**Pull request closed without merging**